src/components/Note.js

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useState } from "react";
+import notesData from "../notesData";
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import Typography from "@material-ui/core/Typography";
+import { useStyles } from "../styles/styles";
+import ActiveNoteDisplay from "./ActiveNoteDisplay";
+
+const Note = (props) => {
+  const classes = useStyles(props);
+  const [activeItem, setActiveItem] = useState(null);
+
+  const PostNote = notesData.map((post) => {
+    return (
```

Comment:
Only one `return` per component! What is `Post`?
Use instead:
```
<div className={classes.root}>
      {!activeItem && <ActiveNoteDisplay />}
      <ItemsNote
        post={post}
        setActiveItem={setActiveItem}
        activeItem={activeItem}
      />
 </div>
```
But what is post? Remove please Items and Post from naming!!!